packages/connectivity/src/scp-cf/destination/destination-accessor-provider-subscriber-lookup.spec.ts

```diff
@@ -326,18 +330,20 @@ describe('JWT type and selection strategies', () => {
 });
 
 describe('call getAllDestinations with and without subscriber token', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     mockServiceBindings();
     mockVerifyJwt();
     mockServiceToken();
+    await destinationCache.clear();
+    await destinationServiceCache.clear();
```

Comment:
[q] Do you really need an await here?

Reply:
You are right, I didn't see that `destinationServiceCache.clear()` didn't return a Promise.